qui/decorators.py

```diff
@@ -90,6 +90,9 @@ def update_tooltip(self,
             if self.vm is None:
                 return
 
+            if self.vm.klass == 'AdminVM':
```

Comment:
For consistency, I'd add some sort of tooltip to dom0 - even if it was just 
"<b>{vmname}</b>\n" \
"Administrative domain.\n"

Reply:
Done!